doc/contributing/docs/api.rst

```diff
@@ -0,0 +1,170 @@
+Documenting the API
+===================
+
+This document contains general guidelines for describing the Tarantool API,
+as well as examples and templates.
+
+Style
+-----
+
+Please write as simply as possible. Describe functionality using short sentences in the present simple tense.
+A short sentence consists of no more than two clauses.
+Consider using `LanguageTool <https://languagetool.org/>`_ or `Grammarly <https://www.grammarly.com/>`_
+to check your English.
+For more style-related specifics, consult the :doc:`Language and style </contributing/docs/style>` section.
+
+..  _contributing-docs-api_version:
+
+Indicating the version
+----------------------
+
+For every new feature or parameter, please indicate the version it was introduced in.
+
+With Tarantool features and parameters, use one of the following Sphinx directives:
+
+..  code-block:: rst
+
+    Since :tarantool-release:`2.10.0`.
+    This is a link to the release notes on GitHub.
+
+    Since :doc:`2.10.0 </release/2.10.0>`.
```

Comment:
I think we need to use only this one in API description. Otherwise, we can get a kind of mess: someone will put a GitHub link and the other one will put a link to documentation RNs. Better to unify this:
- in API section of documentation, we give links to RNs in our documentation
- in RNs, we give links to the original GutHub release notes (:tarantool-release:)